src/script/components/LexicalInput/plugins/ReverseTypeaheadMenuPlugin.tsx

```diff
@@ -0,0 +1,738 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/**
+ * This is a modified version of https://github.com/facebook/lexical/blob/main/packages/lexical-react/src/LexicalTypeaheadMenuPlugin.tsx
+ * It allows showing a list that is filled from the bottom up instead of the top down (the default selected item is at the bottom of the list).
+ * The only differences lie in:
+ * - the default index we select when the component is instanciated (the last one by default)
+ * - a protection to avoid re-rendering the entire component when the list updates (which makes the scroll flicker)
+ */
```

Comment:
reviewing this file can be ignored as it's a duplicate of the native `TypeaheadMenu` from Lexical https://github.com/facebook/lexical/blob/main/packages/lexical-react/src/LexicalTypeaheadMenuPlugin.tsx

Only it handles reversed list (with the selection at the bottom of the list instead of the top)